aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -25,18 +33,21 @@ export const LayerCenterExample = () => {
                 <Box justify="center">
                   <MailOption />
                 </Box>
-                <Heading margin="none" level={2}>
+                <Heading margin="none" level={4}>
```

Comment:
Based on our [Layer guidance](https://design-system.hpe.design/components/layer?q=layer#using-headings-in-layer), I think this should remain as h2 (and adjust the size accordingly). We should update Figma in this case.

Reply:
okay yes Ill let Keeth know if we go that route and change it back!

Reply:
I changed this back let me know if this is okay